samples/Commerce/Commerce.Shared/ViewModels/ProductDetailsViewModel.cs

```diff
@@ -20,9 +20,9 @@ public partial class ProductDetailsViewModel
 		_product = product;
 	}
 
-	public IFeed<Product> Product => Feed.Async(Load);
+	public virtual IFeed<Product> Product => Feed.Async(Load);
 
-	public IFeed<Review[]> Reviews => Product.SelectAsync(async (p, ct) => await _productService.GetReviews(p.ProductId, ct));
+	public virtual IFeed<Review[]> Reviews => Product.SelectAsync(async (p, ct) => await _productService.GetReviews(p.ProductId, ct));
```

Comment:
@dr1rrb is virtual needed here ?

Reply:
Oh, good call - @dr1rrb I needed to make these virtual and override them in the CartProductDetailsViewModel in order to get the code gen to work properly

Reply:
Ok that because you are inheriting one VM from another, the the code gen is not taking that case in consideration yet. That a nice hack to make those properties virtual to work.

Reply:
BTW I just created an issue for that https://github.com/unoplatform/uno.extensions/issues/234